src/builder/QuarkBuilder.sol

```diff
@@ -51,6 +51,170 @@ contract QuarkBuilder {
 
     /* ===== Main Implementation ===== */
 
+    struct BorrowIntent {
```

Comment:
`CometBorrowIntent` to be consistent with `CometSupplyIntent`

Reply:
updated